### PR TITLE
Initial placements findings added

### PR DESCRIPTION
**Pull request by jesseyuen**
Entry added. Eyes please!

### CODE REVIEW

**Comment by jesseyuen** [comment]
[Design History entry: Placements](https://trello.com/c/XM4qODl5/1501-design-history-entry-placements)

**Comment by gregknight1** [comment]
Looking good @jesseyuen

File: app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+Terminology varies by provider with numerous terms being used to describe similar things.
+
+There is the concept of a main location:
+
+* Main school or main setting 
+* Host school 
+* Training location
+
+These locations are typically schoolswhere the SCITT administration is based.
```
Comment (paulrobertlloyd) [inline app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md]:
```suggestion
These locations are typically schools where the SCITT administration is based.
```

File: app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+> “If I’m doing it in August I’ll do “No, I’m doing it later”. If there are any changes I’d have to go back in and amend [them]. If I was keying in September I would put the first. I wouldn’t put the second as there might be changes in the Autumn. [I prefer] keying in the correct information rather than going in and changing it.” 
+> Round 5 P3 (HEI, EY employment)
+
+The rationale behind asking for placements is opaque and can lead to additional frustrations.
+
+> “Compliance states that we need two, but why does the DfE need to know where the person is placed? They just need to know that they are placed?” 
+> Round 6 P6 (SCITT)
+
+We found that further placements are often unknown at the point of requesting TRN.
```
Comment (edwardhorsford) [inline app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md]:
```suggestion
We found that further placements are often unknown at the point of requesting a TRN.
```

File: app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+
+All users feel confident that Register will enable them to change or finalise the placement info at a later point.
```
Comment (edwardhorsford) [inline app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md]:
```suggestion
All users felt confident that Register will enable them to change or finalise the placement info at a later point.
```

File: app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+
+> “I've always keyed in the number of days. I’ve never not. We have our year planned out so to me it doesn't make any difference as long as we can change. I’ve had to change things due to the pandemic.”
```
Comment (edwardhorsford) [inline app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md]:
```suggestion
> “I’ve always keyed in the number of days. I’ve never not. We have our year planned out so to me it doesn't make any difference as long as we can change. I’ve had to change things due to the pandemic.” 
```

File: app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+
+All users feel confident that Register will enable them to change or finalise the placement info at a later point.
+
+> “I've always keyed in the number of days. I’ve never not. We have our year planned out so to me it doesn't make any difference as long as we can change. I’ve had to change things due to the pandemic.” 
+> Round 5 P3 (SCITT)
+
+> “We had a couple of times when a trainee changed placements, I wonder how I would record that? I think I would just add another placement I’m guessing and change the previous placement to just 2 weeks?”
+> Round 5 P6 (SCITT)
+
+We learned that providers’ approach to placements was more varied than we anticipated. We assumed that a trainee would have two linear placements and that terminology would be consistent across the ITT market. Neither turned out to be true and our initial designs did not fit with user needs.
```
Comment (edwardhorsford) [inline app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md]: I wonder if there's a more plain english way of referring to `ITT market`

File: app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+
+### Placement terminology
+
+Terminology varies by provider with numerous terms being used to describe similar things.
+
+There is the concept of a main location:
+
+* Main school or main setting 
+* Host school 
+* Training location
```
Comment (edwardhorsford) [inline app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md]:
```suggestion
* main school or main setting 
* host school 
* training location
```

File: app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+We identified commonly used terms by providers when referring to placements:
+
+* First placement
+* Placement A
+* Second placement
+* Placement B
+* Contrast placement
+* Temporary placement
+* Complementary placement
+* Phase B
```
Comment (edwardhorsford) [inline app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md:77]:
```suggestion
* first placement
* placement A
* second placement
* placement B
* contrast placement
* temporary placement
* complementary placement
* phase B
```

File: app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md [hunk 8]
```diff
@@ -0,0 +1,208 @@
...
+
+Placements are mandatory and provide trainees with  versatile teaching experience. ‘Contrast’ and ‘complementary’ placements reflect that notion and offer the trainee exposure to a different context than their main location, or previous placements. For example, an inner city placement might be contrasted with a rural placement. A placement in an all-girls school might be followed by a placement in an all-boys setting.
```
Comment (edwardhorsford) [inline app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md]:
```suggestion
Placements are mandatory and provide trainees with versatile teaching experience. ‘Contrast’ and ‘complementary’ placements reflect that notion and offer the trainee exposure to a different context than their main location, or previous placements. For example, an inner city placement might be contrasted with a rural placement. A placement in an all-girls school might be followed by a placement in an all-boys setting.
```

File: app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md [hunk 9]
```diff
@@ -0,0 +1,208 @@
...
+The current design displays placements in order of them being added to Register. If a trainee changed locations midway through their placement, the update will currently not appear in chronological order.
+
+>“There are instances where the trainee doesn't suit the school. Looking at this I don't know how [to reflect that in the record] unless there is another option to add another school there could be two or three schools needed in the for instance.”  
+>Round 5 P5 (SCITT)
+
+## Next steps
+
+### Iterate our designs
+
+Requiring the user to input  a specific start date and duration in weeks doesn't reflect how placements really happen and therefore not meet some of our user’s needs.
```
Comment (edwardhorsford) [inline app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md]:
```suggestion
Requiring the user to input a specific start date and duration in weeks doesn’t reflect how placements really happen and therefore not meet some of our user’s needs. 
```

File: app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md [hunk 10]
```diff
@@ -0,0 +1,208 @@
...
+
+### Consult data consumers
+ 
+We need to better understand why our data consumers want to know this data/what exactly they are tracking. If they are interested in understanding how much time the trainees spend physically teaching in schools (as opposed to being trained) - our results might be skewed. 
+
+## Design todos
+
+* include a search that allows for school name, postcode or URN
+* if we decide to pursue collect dates, we will also need to collect the year the placement started
+* give our users' confidence by adding a location name, URN and postcode on the confirmation page
```
Comment (edwardhorsford) [inline app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md]:
```suggestion
* give our users’ confidence by adding a location name, URN and postcode on the confirmation page
```

File: app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md [hunk 11]
```diff
@@ -0,0 +1,208 @@
...
+## Concerns with collecting placement details
+
+We have concerns about the implications of making the collection of placement details a required task.
+ 
+Considering the frequency of location change for some trainees, our hunch is that this will:
+
+* create a significant amount of additional work, becoming a burden for our users
+* increase errors as users might forget about the updating the system with placement changes
+
+By acknowledging the scale of the burden we are knowingly compromising our users' experience.
```
Comment (edwardhorsford) [inline app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md]:
```suggestion
By acknowledging the scale of the burden we are knowingly compromising our users’ experience.
```

File: app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md [hunk 12]
```diff
@@ -0,0 +1,208 @@
...
+
+How might we make the process of adding placement data fit with the reality of how placements happen to reduce effort and minimise potential for error?
+
+Asking for the placement data at trainee record setup allows for flexibility and for users to create a workflow that suits them. This creates additional complexity as not all placements are known at the time of record setup so in this scenario our users need to understand that they can do some of a task now, but will need to finish it off later.
+
+In turn collecting placement details at time of QTS recommendation removes complexity when creating records yet would mean users spend significantly more time completing the task of recommending for qualifications.
+
+By exploring different ways of prompting outstanding actions we could change provider's behavior to record data changes as soon as they occur, minimising potential for error and reducing time needed to recommend a trainee for QTS.
```
Comment (edwardhorsford) [inline app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md]:
```suggestion
By exploring different ways of prompting outstanding actions we could change provider’s behaviour to record data changes as soon as they occur, minimising potential for error and reducing time needed to recommend a trainee for QTS.
```

**Review by edwardhorsford — CHANGES_REQUESTED** [review]

File: app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md (hunk 5 above)
Comment (gregknight1) [reply app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md]: ...... and that ITT provider terminology would be consistent.

Comment (jesseyuen) [reply app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md]: Agreed. Gregs recommendation added.

**Review by edwardhorsford — APPROVED** [review]

**Pull request merged**